app/src/main/java/by/bsuir/playgame/Activity/PlacementRoomActivity.java

```diff
@@ -0,0 +1,230 @@
+            if (Integer.parseInt(textView1.getText().toString()) > 0)
+                shipViewModel.setShip("1");
+            else
+                Toast.makeText(this, "Выберите другой тип корабля", Toast.LENGTH_SHORT).show();
+
+        });
+        button2.setOnClickListener(item -> {
+            if (Integer.parseInt(textView2.getText().toString()) > 0)
+                shipViewModel.setShip("2");
+            else
+                Toast.makeText(this, "Выберите другой тип корабля", Toast.LENGTH_SHORT).show();
+
+        });
+        button3.setOnClickListener(item -> {
+            if (Integer.parseInt(textView3.getText().toString()) > 0)
+                shipViewModel.setShip("3");
+            else
+                Toast.makeText(this, "Выберите другой тип корабля", Toast.LENGTH_SHORT).show();
+
+        });
+        button4.setOnClickListener(item -> {
+            if (Integer.parseInt(textView4.getText().toString()) > 0)
+                shipViewModel.setShip("4");
+            else
+                Toast.makeText(this, "Выберите другой тип корабля", Toast.LENGTH_SHORT).show();
+
+        });
+
+        buttlebutton.setOnClickListener(item -> {
+            if (Integer.parseInt(textView1.getText().toString()) +
+                    Integer.parseInt(textView2.getText().toString()) +
+                    Integer.parseInt(textView3.getText().toString()) +
+                    Integer.parseInt(textView4.getText().toString()) == 0) {
+                Toast.makeText(this, "В бой!", Toast.LENGTH_SHORT).show();
+
+                messageRef.addListenerForSingleValueEvent(new ValueEventListener() {
+                    @Override
+                    public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
+                        if (Objects.requireNonNull(dataSnapshot.getValue()).toString().equals("Placement Start")) {
+                            progressDialog.setMessage("Please wait second player");
+                            progressDialog.show();
+                            progressDialog.setCanceledOnTouchOutside(false);
+                            message = "Waiting second player";
+                        } else {
+                            message = "Start Game";
+                        }
+                        messageRef.setValue(message);
+                    }
+
+                    @Override
+                    public void onCancelled(@NonNull DatabaseError error) {
+
+                    }
+                });
+            } else {
+                Toast.makeText(this, "Раставленны не все корабли", Toast.LENGTH_SHORT).show();
+            }
+
+        });
+
+        button5.setOnClickListener(item -> {
+            if (Integer.parseInt(textView1.getText().toString()) +
+                    Integer.parseInt(textView2.getText().toString()) +
+                    Integer.parseInt(textView3.getText().toString()) +
+                    Integer.parseInt(textView4.getText().toString()) != 10) {
```

Comment:
```suggestion
                    Integer.parseInt(textView4.getText().toString()) != MAX_SHIP_COUNT) {
```

'Magic' numbers are bad!